samples/GraphQL.DataLoader.Sample.Default/DealershipDbContext.cs

```diff
@@ -0,0 +1,90 @@
+using GraphQL.DataLoader.Di.Sample.Types;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+
+namespace GraphQL.DataLoader.Di.Sample;
+
+public class DealershipDbContext : DbContext
+{
+    public DbSet<Car> Cars => Set<Car>();
+    public DbSet<Salesperson> Salespeople => Set<Salesperson>();
+
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+    {
+        optionsBuilder
+            .UseSqlite(
+                new SqliteConnectionStringBuilder()
+                {
+                    Mode = SqliteOpenMode.ReadWriteCreate,
+                    BrowsableConnectionString = true,
+                    DataSource = "my.db"
+                }
+                .ToString()
+            )
+            .UseSeeding((ctx, _) =>
+            {
+                if (ctx.Set<Salesperson>().Any())
+                    return;
+                ctx.Set<Salesperson>().AddRange(_seedSalespeople);
+                ctx.SaveChanges();
+            })
```

Comment:
duplicate
```suggestion
```

Reply:
Actually in their Docs they recommend implementing both UseSeeding and UseSeedingAsync together.

>  When using this feature, it is recommended to implement both UseSeeding and UseAsyncSeeding methods using similar logic, even if the code using EF is asynchronous.

Reply:
:+1: